src/Service/LibraryService.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace OpenBibIdApi\Service;
+
+class LibraryService extends Service implements LibraryServiceInterface
```

Comment:
Niet nodig om ook nog eens "implements LibraryServiceInterface" toe te voegen; Is reeds gebeurd door de Service class.